python/src/scipp/html/formatting_html.py

```diff
@@ -382,7 +382,7 @@ def _mapping_section(mapping,
                      details_func=None,
                      max_items_collapse=None,
                      enabled=True):
-    n_items = len(mapping) if hasattr(mapping, "__len__") else 1
```

Comment:
What does this change? Does it use the normal `__str__` of Python containers as opposed to formatting the elements ourselves?

Reply:
Detection of `__len__` was used here to figure out if this is a dataset or meta data dict, or something else. Now `DataArray` also has `__len__`, but we don't want to trigger the "dict formatting" code.

Reply:
Ok makes sense.